.github/workflows/benchmark.yml

```
@@ -0,0 +1,71 @@
+        run: |
+          {
+            echo 'RESULT<<EOF'
+            cat benchmark/result.md
+            echo EOF
+          } >> "$GITHUB_OUTPUT"
+
+      - name: Post PR comment
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-tag: hyperfine-benchmark
```

actionlint (1.7.12)

[error] input "comment-tag" is not defined in action "peter-evans/create-or-update-comment@v4". available inputs are "append-separator", "body", "body-file", "body-path", "comment-id", "edit-mode", "issue-number", "reactions", "reactions-edit-mode", "repository", "token"

(action)